README.md

```diff
@@ -78,10 +78,12 @@ log.Println("PREDICTION: ", out[rt.Outputs[0].Name])
 ```
 
 ### Cleanup
-Runtimes are by necessity, memory unsafe. In order to prevent memory leaks, the Cleanup function should be deferred to whenever creating a new Runtime. If this isn't called, there will be memory leaks from the CGo calls. This should typically be called with "defer", to avoid leaks if there is a panic or fatal error.
+Runtimes are, by necessity, memory unsafe. The Cleanup function should be deferred to prevent memory leaks whenever creating a new Runtime. If this isn't called, there will be memory leaks from the CGo calls. This should typically be called with "defer," to avoid leaks if there is a panic or fatal error.
+
 ## Testing
 This package uses the go testing package. The tests are stored in the /tests/ folder and can be called with "go test -run ^TestToRun ./relative/path/to/tests"
 
->Note: Tests utilize other dependencies, such as github.com/sugarme/tokenizer and github.com/galeone/tfgo
+>Note: Tests utilize other dependencies, such as github.com/sugarme/tokenizer and github.com/galeone/tfgo.
+
 ### Models
 Some models are not included in this repository. For the mnist-8 or roberta-sequence-classification-9 models, see the [ONNX Model Zoo](https://github.com/onnx/models). The boston_housing_regression, scaler, and string-convert models are included.
```

Comment:
Just realized that we don't have the exact link to the models. @johntrob14 Can you please update this and add the exact link for mnist-8 and roberta model?